tests/integration/cqlengine/model/test_udts.py

```diff
@@ -286,3 +286,43 @@ class AllDatatypesModel(Model):
 
         for i in range(ord('a'), ord('a') + 15):
             self.assertEqual(input[chr(i)], output[chr(i)])
+
+    def test_nested_udts_inserts(self):
+        """
+        Test for inserting collections of user types using cql engine.
+        test_nested_udts_inserts Constructs a model that contains a list of usertypes. It will then attempt to insert
```

Comment:
There should be a newline between the brief description and the full description.